### PR TITLE
Add documentation for the new Husqvarna Automower integration

### PR DESCRIPTION
**Pull request by Thomas55555**
## Proposed change

Add documentation for the new Husqvarna Automower integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/109073
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4655
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/husqvarna_automower.markdown [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+The My Home Assistant redirect feature needs to be setup to redirect to your Home Assistant installation. See [My FAQ](https://my.home-assistant.io/faq) for additional information.
+
+1. Add the integration to your Home Assistant installation and test the redirect feature by following below link:
+   [![my_button](https://my.home-assistant.io/badges/config_flow_start.svg)](https://my.home-assistant.io/redirect/config_flow_start/?domain=husqvarna_automower)
+
+2. Acknowledge prompts to open link, install Husqvarna Automower integration.
+
+3. Acknowledge prompt to setup application credentials.
+
+4. Enter the following from the Husqvrana developer tab:
```
Comment (pascalwinters) [inline source/_integrations/husqvarna_automower.markdown]: Typo in `Husqvrana`

File: source/_integrations/husqvarna_automower.markdown [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+3. Create a new application:
+   - Name is required but can be anything, for example "My Home Assistant"
+
+   - Description is optional
+
+   - Redirect URL: `https://my.home-assistant.io/redirect/oauth`
+     Make sure no extra spaces were appended at end of URL from copy and paste.
+
+    ![Creat new Application](https://github.com/Thomas55555/home-assistant.io/assets/59625598/97abbbac-a089-4d9a-be3d-3a3f9c50442c)
```
Comment (pascalwinters) [inline source/_integrations/husqvarna_automower.markdown]: Typo in `Creat`

Comment (joostlek) [reply source/_integrations/husqvarna_automower.markdown]:
```suggestion
    ![Create new Application](https://github.com/Thomas55555/home-assistant.io/assets/59625598/97abbbac-a089-4d9a-be3d-3a3f9c50442c)
```

Comment (joostlek) [inline source/_integrations/husqvarna_automower.markdown]: Please don't use your own assets in the docs :)

File: source/_integrations/husqvarna_automower.markdown (hunk 1 above)
Comment (djpremier) [inline source/_integrations/husqvarna_automower.markdown]:
```suggestion
4. Enter the following from the Husqvarna developer tab:
```

**Review by djpremier — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

File: source/_integrations/husqvarna_automower.markdown [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+ha_integration_type: integration
+ha_domain: husqvarna_automower
+---
+
+The Husqvarna Automower integration provides connectivity with Husqvarna Automowers lawn mowers through Husqvarna's cloud API. Only mowers with *Automower® Connect* or with the *Automower® Connect Module* are supported.
+
+In order to use this integration you must properly configure OAuth2 credentials using your Husqvarna account.  Refer to [this guide](https://developer.husqvarnagroup.cloud/docs/get-started) for general overview of the process.
+Your Husqvarna account username/password used for the *Automower® Connect*  phone app is required.  Most users probably created a Husqvarna account during initial mower setup.
+
+1. Go to the [Husqvarna Developers Portal](https://developer.husqvarnagroup.cloud) and sign in with your Husqvarna account. Authorize *Developer Portal* to access Husqvarna account when prompted.
```
Comment (MartinHjelmare) [inline source/_integrations/husqvarna_automower.markdown]:
```suggestion
1. Go to the [Husqvarna Developer Portal](https://developer.husqvarnagroup.cloud) and sign in with your Husqvarna account. Authorize *Developer Portal* to access Husqvarna account when prompted.
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]
Have you made sure to invalidate the example application credentials shown in the image?

**Comment by Thomas55555** [comment]
> Have you made sure to invalidate the example application credentials shown in the image

Thanks for the hint. Yes, it's invalidated.

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**